content/applications/essentials/activities.rst

```diff
@@ -3,236 +3,332 @@ Activities
-  planned activity in the chatter.
-- If either :guilabel:`Phonecall` or :guilabel:`Meeting` are selected, users have the option to open
-  their calendar to schedule a time for this activity.
-- If :guilabel:`Request Signature` is selected, a link is added to the planned activity in the
-  chatter that opens a signature request pop-up window.
+When all the fields are completed. click :guilabel:`Request` to send the document request.
 
-.. note::
-   The actions available to select on an activity type vary, depending on the applications currently
-   installed in the database.
+.. image:: activities/request-doc.png
+   :align: center
+   :alt: The Request a file form, with all fields filled out to request a contract.
 
-Default user
-~~~~~~~~~~~~
+.. _activities/types:
 
-To automatically assign this activity to a specific user when this activity type is scheduled,
-choose a name from the :guilabel:`Default User` drop-down menu. If this field is left blank, the
-activity is assigned to the user who creates the activity.
+Activity types
+==============
 
-Default summary
-~~~~~~~~~~~~~~~
+To view the currently configured types of activities in the database, navigate to
+:menuselection:`Settings app --> Discuss --> Activities --> Activity Types`.
 
-To include notes whenever this activity type is created, enter them into the :guilabel:`Default
-Summary` field.
+.. image:: activities/settings-activities-types.png
+   :align: center
+   :alt: Activity Types button in the Settings application under the Discuss section.
 
-.. note::
-   The information in the :guilabel:`Default User` and :guilabel:`Default Summary` fields are
-   included when an activity is created. However, they can be altered before the activity is
-   scheduled or saved.
+Doing so reveals the :guilabel:`Activity Types` page, where the existing activity types are found.
 
-Next activity
--------------
+.. image:: activities/activity-list.png
+   :align: center
+   :alt: The list of activity types already configured and available.
 
-To automatically suggest, or trigger, a new activity after an activity has been marked complete, the
-:guilabel:`Chaining Type` must be set.
+Edit activity types
+-------------------
 
-Suggest next activity
-~~~~~~~~~~~~~~~~~~~~~
+To edit an existing :ref:`activity type <activities/types>`, click on the activity type, and the
+:guilabel:`Activity Type` form loads.
 
-In the :guilabel:`Chaining Type` field, select :guilabel:`Suggest Next Activity`. Upon doing so, the
-field underneath changes to: :guilabel:`Suggest`. Click the :guilabel:`Suggest` field drop-down menu
-to select any activities to recommend as follow-up tasks to this activity type.
+Make any desired changes to the :guilabel:`Activity Type` form. The form automatically saves, but it
+can be saved manually at any time by clicking the *save manually* option, represented by a
+:guilabel:`(cloud upload)` icon, located in the top-left corner of the page.
```

Comment:
I htink I will leave it because the section is so short,  s it doesn't feel like too much 'extra' ifo (and it applies to all the forms, so I thik it's OK to keep it in here).